package.json

```diff
@@ -4,27 +4,32 @@
   "private": true,
   "scripts": {
     "dev": "next dev --turbopack",
-    "build": "next build",
+    "build": "prisma generate && next build",
```

Comment:
can we add a separate script for generating the prisma client `prisma:generate` and then use that in the build script - `"build": "npm run prisma:generate && next build`